Encrypt-S/kauri-wallet
# Create PWA

Add PWA / Service Worker / Offline support.

This just adds the most basic implementation, as well as temp banner at the bottom asking you to refresh if an update is found. (You will not see this expect in production, and when an update has come through).

### Testing
- Open netlify build. 
- Open dev tools. 
- Goto Network Tab
- Select offline
- Reload page. 
- See that it still loads (and network shows content coming from service workers).
